### DISCUSSION TITLE
Copy Settings to another user

### DISCUSSION THREAD

**Discussion by Bammede** [prasanthrangan/hyprdots#857] [Q&A]
Hyperdots runs on the one user account. How second user can use hyperdots with settings of the first user? just copy & paste directories from Home dir?

**Comment by kRHYME7** [comment]
Hello you can use ``` Hyprdots backup all ``` command to backup current listed configs.  Then navigate that backup inside ```  ~/.config/cfg_backups/* ```
This folder will be backed up inside  you can use this to save all the configs. 

Now on the second user you still need to install all Hyprdots related installation.  So that the Needed script will be set. 
Now if you want to restore that backup config to a new user you need to run 
``` Hyprdots restore Clone /path/to/the/copied/backup ``` then read the instructions on what to preserve/Overwrite



So if you decide not to do the installation again then manually copying of files  won't  be very helpful as some symbolic links are  named to the 1st user.
